kernel/src/fs/ramfs/fs.rs

```diff
@@ -106,7 +106,7 @@ struct RamInode {
 
 struct Node {
     inner: Inner,
-    metadata: InodeMeta,
+    metadata: SpinLock<InodeMeta>,
```

Comment:
This approach seems to bring benefits in case when we touch single field mostly, while actually we often touch multiple fields, like changing timestamps. Another example is applications issue a lot of `sys_stat` that requires touching all fields of `InodeMeta`. In these cases one `SpinLock::lock()` seems cheaper than lots of atomic loads/stores. So I think this optimization can wait. (Also we don't have a `AtomicDuration` for now)